packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1470,6 +1492,7 @@ class _WidgetsAppState extends State<WidgetsApp> with WidgetsBindingObserver {
     } else {
       assert(_navigator != null);
       routing = Navigator(
+        restorationScopeId: 'nav',
```

Comment:
why "nav"? Why not "n" or "navigator" or "\0x01" or "$Navigator" or something?

Reply:
Yes. Any of those are valid as long as they are unique within the surrounding  restoration scope. Here, the navigator is the only thing in this scope. Children (the individual routes) have their own scope.

Reply:
So why "nav" specifically?